### ISSUE TITLE
Typo in the documentation

### ISSUE THREAD

**Issue by KES777** [Perl5-FFI/FFI-Platypus#199]
https://metacpan.org/pod/FFI::Platypus::Type#opaque-pointers

>$ffi->attach('archive_read_new => [] => 'archive');

should be: `$ffi->attach('archive_read_new' => [] => 'archive');`

missed quote

**Comment by KES777** [comment]
if you do not mind I will put different typos into this issue.

https://metacpan.org/pod/FFI::Platypus::Type#Strings
``` 
typedef const char *(*get_message_t)(void);
void
print_message(get_message_t get_message)
{
  const char *str;
  str = get_message();
  printf("message = %s\n");
}
```
Code seems not complete. Do you forget to provide `str` parameter while `printf`?

**Comment by KES777** [comment]
https://metacpan.org/pod/FFI::Platypus::Type#Pointer-/-References

> In Perl the same is task is 

just typo `is`

**Comment by plicease** [comment]
Thank you for this report fixed in 1.01.
